src/renderer/components/notification-comp.tsx

```diff
@@ -53,8 +53,8 @@ export default class NotificationComp extends React.Component<{}, IState> {
      */
     public render(): JSX.Element {
         const { title, company, body, image, icon, id, color } = this.state;
-        const colorHex = color ? '#' + color : undefined;
-        const isLightTheme = colorHex ? colorHex.match(whiteColorRegExp) : true;
+        const colorHex = (color && !color.startsWith('#')) ? '#' + color : color;
```

Comment:
Is this check because we don't get `#` with colour codes from SFE at times?

Reply:
Yes, that's correct